static/js/lib/learning_resources.js

```diff
@@ -27,24 +32,130 @@ export const availabilityLabel = (availability: ?string) => {
   }
 }
 
+export const availabilityFilterToMoment = (filter: string, ending: boolean) => {
+  // Convert an Elasticsearch date_range filter string to a moment
```

Comment:
can you add an explanation here of why it returns `undefined` for the availability now filter? it's a confusing detail that would be hard to get from just the code itself so it's good to explain I think

Reply:
✔️